src/memory_region.rs

```diff
@@ -155,20 +147,165 @@ pub enum AccessType {
     Store,
 }
 
-/// Indirection to use instead of a slice to make handling easier
+impl fmt::Display for AccessType {
```

Comment:
Why implement `Display` manually here if we could just derive `Debug`?

Reply:
I honestly can't remember at all why I added this 😂 But tests pass without so I nuked it